pkg/batchd/cache/cache.go

```diff
@@ -146,6 +152,24 @@ func newSchedulerCache(config *rest.Config) *SchedulerCache {
 			},
 		})
 
+	// create informer for priority class information
+	sc.pcInformer = informerFactory.Scheduling().V1alpha1().PriorityClasses()
```

Comment:
We do not need to cache `PriorityClass`, admission controller will help to translate `PriorityClass` to priority; we just need to handle priority here.